src/components/Navigation.js

```diff
@@ -1,35 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './navigation.css';
 import { NavLink } from 'react-router-dom';
+import Hamburger from 'hamburger-react';
 import logo from '../images/icons/logo.png';
 
-const Navigation = () => (
-  <div>
-    <nav className="navigation">
-      <section className="logo-div">
-        <img className="logo-image" src={logo} alt="Maths Magician Logo" />
-        Maths Magician
-      </section>
-      <section>
-        <ul className="nav-ul">
-          <li>
-            <NavLink className="nav-link" to="/">
-              Home
-            </NavLink>
-          </li>
-          <li>
-            <NavLink className="nav-link" to="/calculator">
-              Calculator
-            </NavLink>
-          </li>
-          <li>
-            <NavLink className="nav-link" to="/quotes">
-              Quotes
-            </NavLink>
-          </li>
-        </ul>
-      </section>
-    </nav>
-  </div>
-);
+function Navigation() {
+  const [isOpen, isClosed] = useState(false);
+  return (
+    <div>
+      <nav className="navigation">
+        <section className="logo-div">
+          <img className="logo-image" src={logo} alt="Maths Magician Logo" />
+          Maths Magician
+        </section>
```

Comment:
Good Idea to wrap your navigation inside the `<nav>` tag. Additionally, I would like to suggest wrapping your `<nav>` inside a `<header>` tag.
Also here is not the proper place to add the `<section>` tag, because sections are self-descriptive containers with their own `header`, so wrapping those elements using a `<div>` tag would be a better idea.